[CN-448] Update the tutorial for showing the enhanced use case of WanReplication (#2)
hazelcast-guides/hazelcast-platform-operator-wan-replication: docs/modules/ROOT/pages/hazelcast-platform-operator-wan-replication.adoc

- Deploy two Hazelcast clusters.

- Create two Hazelcast map configurations on one of the clusters.

- Synchronize map data between the two Hazelcast clusters.

== Before you Begin

Before starting this tutorial, make sure that you have the following:

* A running https://kubernetes.io/[Kubernetes] cluster
* The Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
* A deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]

== Step 1. Start the Hazelcast Cluster

> 

Reviewer: In this step, maybe you could just have the instruction below and remove the instruction on line 31.
```suggestion
. Create a secret with your link:http://trialrequest.hazelcast.com/[Hazelcast Enterprise License].
```

- Deploy two Hazelcast clusters.

> - Create two Hazelcast map configuration in one of the cluster.

Reviewer: ```suggestion
- Create two Hazelcast map configurations on one of the clusters.
```

> - Synchronize map data between two Hazelcast clusters.

Reviewer: ```suggestion
- Synchronize map data between the two Hazelcast clusters.
```
Also suggestions for the Before you Begin section below:

Before starting this tutorial, make sure that you have the following:
* A running https://kubernetes.io/[Kubernetes] cluster
* The Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[`kubectl`]
* A deployment of Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]

----
+

> . Check the status of the clusters to make sure that it's running.

Reviewer: ```suggestion
. Check the status of the clusters to make sure that both clusters are running.
```

> == Step 2. Create WAN Replication Configuration

Reviewer: ```suggestion
== Step 2. Create a WAN Replication Configuration
```

> . Create two map in the first cluster.

Reviewer: ```suggestion
. Create two maps on the first cluster. In this example, the following maps are created:

- `example-map-1`
- `example-map-2`.
```

+
[source, shell]
----
cat <<EOF | kubectl apply -f -
apiVersion: hazelcast.com/v1alpha1
kind: Map
metadata:
  name: example-map-1
spec:
  hazelcastResourceName: hazelcast-first
---
apiVersion: hazelcast.com/v1alpha1
kind: Map
metadata:
  name: example-map-2
spec:
  hazelcastResourceName: hazelcast-first
EOF
----

> . Create a WAN Replication Configuration.

Reviewer: ```suggestion
. Create the configuration for WAN replication:
```

+

> - Use the first cluster as the source cluster by giving the cluster name as a resource to WAN Replication configuration.

Reviewer: ```suggestion
- Use the first cluster as the source cluster by adding its name as a resource in the WAN Replication configuration.
```

> Giving the cluster name as a resource starts the WAN Replication for both of the maps created earlier.

Reviewer: ```suggestion
Adding the cluster name as a resource starts WAN replication for both the maps that you created earlier.
```

+

> - Use the second cluster as the target cluster to receive the WAN Replication events.

Reviewer: ```suggestion
- Add the second cluster as the target cluster to receive the WAN Replication events.
```

+

> Run the following command to configure the replication

Reviewer: ```suggestion
Run the following command to apply the configuration.
```

--

====

> . Now start the application for each map with their name as a argument to fill the maps.

Reviewer: ```suggestion
. Now start the application for each map, using the map name as an argument to fill each map with random entries. If you're reusing the sample code from this tutorial, use the map names `example-map-1` and `example-map-2`.
```

+

> Map Names -> example-map-1, example-map-2

Reviewer: ```suggestion

```
Replaced by suggestion above.

Current map size: 10
----

> == Step 3. Verify that the Maps' Entries were Replicated

Reviewer: ```suggestion
== Step 3. Verify the Replication of Map Entries
```

> In this step, you verify the sizes of the maps on the second cluster to make sure that they received all the WAN Replication events.

Reviewer: ```suggestion
In this step, you'll check the sizes of the maps on the second, target cluster to make sure that WAN replication events have been received.
```

> . Configure the Hazelcast client to connect to the second cluster as you did in <<configure-client, Configure the Hazelcast Client>>.

Reviewer: ```suggestion
. Configure the Hazelcast client to connect to the second cluster, as you did in <<configure-client, Configure the Hazelcast Client>>.
```

----
--
====

> . Start the application for each map with their name as a argument to check the map size and see if the WAN Replication was successful.

Reviewer: ```suggestion
. Start the application for each map, using the map name as an argument to check the map size, and to check that WAN replication was successful. If you're reusing the sample code from this tutorial, use the map names `example-map-1` and `example-map-2`.
```

+

> Map Names -> example-map-1, example-map-2

Reviewer: ```suggestion

```
Included text in bullet point above.

[source, shell]
----
Successful connection!
Current map (<MAP-NAME>) size: 12
----

> == Clean Up

Reviewer: Suggested updates to line 424:

To remove all custom resources and PVCs, run the following commands:
